test.js

```diff
@@ -5,6 +5,17 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const expect = require('chai').expect;
 const sanitize = require('./index.js');
+const callTracker = () => {
```

Comment:
Node.js 12+ has https://nodejs.org/api/assert.html#assert_class_assert_calltracker
But Node.js 10 does not this.